packages/graph-editor/src/registry/inputControls.tsx

```diff
@@ -8,11 +8,10 @@ import {
   Select,
   Stack,
 } from '@tokens-studio/ui';
+import properties from 'mdn-data/css/properties.json' with { type: 'json' };
```

Comment:
I think we need to test with nextjs. We're currently using `transpilePackages` to force the nextjs server to transpile the ouput from the engine and editor as this was one of the consumers that threw issues with the original assert

Reply:
as long as it's using the latest bundler / tools etc. they should support import attributes, but it might be necessary to bump some stuff there if it doesn't.